addon/components/detail-learning-materials.js

```diff
@@ -82,6 +83,18 @@ export default class DetailCohortsComponent extends Component {
     this.displayAddNewForm = true;
   }
 
+  @action
+  closeLearningmaterialManager() {
+    this.managingMaterial = null;
+    scrollIntoView(this.title);
```

Comment:
I think 
```
    scrollIntoView(this.title, {
      align: { top: 0 }
    });
```

will put this [at the top of the page](https://www.npmjs.com/package/scroll-into-view).

Reply:
i stand corrected! :+1:

Reply:
:heavy_check_mark: